syserol/figures/figure10.py

```diff
@@ -41,16 +54,27 @@ def makeFigure():
             "Groups": subjinfo["class.etuv"],
         }
         df = pd.DataFrame(data)
-        a = sns.scatterplot(
-            x=f"Component {i+1} Measurement",
-            y=f"Component {i+2} Measurement",
-            hue="Groups",
-            data=df,
-            palette="Set1",
-            legend="brief",
-            ax=ax[j],
-        )
-        a.legend(loc="center left", bbox_to_anchor=(1, 0.5), ncol=1)
+        if j == 4:
+            a = sns.scatterplot(
+                x=f"Component {i+1} Measurement",
+                y=f"Component {i+2} Measurement",
+                hue="Groups",
+                data=df,
+                palette="Set1",
+                legend="brief",
+                ax=ax[j],
+            )
+            a.legend(loc="center left", bbox_to_anchor=(1, 0.5), ncol=1)
+        else:
+            a = sns.scatterplot(
+                x=f"Component {i+1} Measurement",
+                y=f"Component {i+2} Measurement",
+                hue="Groups",
+                data=df,
+                palette="Set1",
+                legend=False,
```

Comment:
Rather than duplicate this whole block you can do `legend = "brief" if j == 4 else False`.

Reply:
Thanks, cool trick!